libraries/botbuilder/package.json

```diff
@@ -38,6 +38,7 @@
     "filenamify": "^4.1.0",
     "fs-extra": "^7.0.1",
     "htmlparser2": "^6.0.1",
+    "runtypes": "6.3.0",
```

Comment:
Should be `~6.3.0`

Reply:
:( I actually checked for that but must've pulled up the wrong package.json